app/controllers/api/v1/accounts/contacts_controller.rb

```diff
@@ -31,10 +31,10 @@ def contact_params
   end
 
   def fetch_contact
-    @contact = current_account.contacts.find(params[:id])
+    @contact = Current.account.contacts.find(params[:id])
   end
 
   def contact_create_params
-    params.require(:contact).permit(:account_id, :inbox_id).merge!(name: SecureRandom.hex)
+    params.require(:contact).permit(:name, :email, :phone_number)
```

Comment:
Are we fully removing the `inbox_id` from contact?

Reply:
inbox_id is leftover from our old code :D

Reply:
Since the contact model does not have `inbox_id` anymore, this is not a valid concern.